src/compiler/scanner.rs

```diff
@@ -416,6 +441,27 @@ impl Scanner {
         None
     }
 
+    pub fn re_scan_template_token(
+        &self,
+        on_error: Option<ErrorCallback>,
+        is_tagged_template: bool,
+    ) -> SyntaxKind {
+        Debug_.assert(
+            self.token() == SyntaxKind::CloseBraceToken,
+            Some("'reScanTemplateToken' should only be called on a '}'"),
```

Comment:
Maybe this should use `'re_scan_template_token'` instead of `'reScanTemplateToken'`?